api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -364,7 +378,7 @@ default Span recordException(Throwable exception) {
    * <p>Use this method for specifying explicit end options, such as end {@code Timestamp}. When no
    * explicit values are required, use {@link #end()}.
    *
-   * @param timestamp the explicit timestamp from the epoch, for this {@code Span}. {@code 0}
+   * @param timestamp the explicit timestamp from the epoch, for this {@code Span}. {@code null}
    *     indicates current time should be used.
```

Comment:
Careful about this change.

Reply:
This was about the special behavior for the value `0` which we need to keep. If following the above suggestion, we don't need to add a special note about `null` here (behavior is undefined, so "current time" is a fine behavior).

Reply:
Why is 0 value special?

Reply:
`0` is only special for the method above. Not an issue for this method which takes an `Instant`, I don't think.